src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,118 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService;
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+        user.setPassword("111111");
+        user.setAge(20);
+        user.setLogin("test_user@gmail.com");
+    }
+
+    @Test
+    void register_userAlreadyExistInStorage() {
+        User existingUser = new User();
+        existingUser.setPassword("111111");
+        existingUser.setAge(20);
+        existingUser.setLogin("test_user@gmail.com");
+        Storage.people.add(existingUser);
+        assertThrows(RuntimeException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void ageIsNull_notOk() {
```

Comment:
Method's name should be first. Fix it in all places
```suggestion
    void register_ageIsNull_notOk() {
```